WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.kt

```diff
@@ -689,7 +690,6 @@ class MeFragment : Fragment(R.layout.me_fragment), OnScrollToTopListener {
     fun onEventMainThread(event: GravatarUploadFinished) {
         binding?.showGravatarProgressBar(false)
         if (event.success) {
-            AnalyticsTracker.track(ME_GRAVATAR_UPLOADED)
```

Comment:
Just a note here, I removed this one because `ME_GRAVATAR_UPLOADED` is already tracked on line 676